LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -122,8 +122,7 @@ fast/events/detect-caps-lock.html [ Pass ]
 fast/forms/auto-fill-button/caps-lock-indicator-should-not-be-visible-when-auto-fill-strong-password-button-is-visible.html [ Pass ]
 fast/forms/password-scrolled-after-caps-lock-toggled.html [ Pass ]
 
-# previously: webkit.org/b/194170
-webkit.org/b/260974 [ Release ] fast/repaint/placeholder-after-caps-lock-hidden.html [ Pass Failure Timeout ]
+[ Release ] fast/repaint/placeholder-after-caps-lock-hidden.html [ Pass ]
```

Comment:
Is `Release` necessary? (I don't think so?)

Reply:
Right, it probably isn't. Let me verify in `Debug` too and then I'll remove it

Reply:
`Release` is indeed unnecessary, removed in https://github.com/WebKit/WebKit/commit/95499b447b76f36b5e781c5b4bbe5aa2fe22fed0